storage/store/test.go

```diff
@@ -46,10 +62,49 @@ func handleGetError(t *testing.T, err error, found bool) {
 	}
 }
 
+// NewTestConsulInstance allows to provide new Consul instance for tests
+// This is a private Consul server instantiation as done in github.com/ystia/yorc/v4/testutil
+// This allows avoiding cyclic dependencies with deployments store package
+func NewTestConsulInstance(t testing.TB) (*testutil.TestServer, *api.Client, config.Configuration) {
+	logLevel := "debug"
+	if isCI, ok := os.LookupEnv("CI"); ok && isCI == "true" {
+		logLevel = "warn"
+	}
+
+	cb := func(c *testutil.TestServerConfig) {
+		c.Args = []string{"-ui"}
+		c.LogLevel = logLevel
+	}
+
+	srv1, err := testutil.NewTestServerConfig(cb)
+	if err != nil {
+		t.Fatalf("Failed to create consul server: %v", err)
+	}
+
+	workingDir, err := ioutil.TempDir("/tmp", "work")
```

Comment:
Same as above consider using `./testdata/` instead of `/tmp`

Reply:
ok

Reply:
Fixed